src/OSVRTrackedDevice.cpp

```diff
@@ -384,9 +491,12 @@ float OSVRTrackedDevice::GetFloatTrackedDeviceProperty(vr::ETrackedDevicePropert
             *error = vr::TrackedProp_ValueNotProvidedByDevice;
         return default_value;
     case vr::Prop_DisplayFrequency_Float: // TODO
-        if (error)
-            *error = vr::TrackedProp_ValueNotProvidedByDevice;
-        return default_value;
+		// Copyright Razer LLC 2016
+		// HARDCODED
+		// This needs to be read from the Render Manager Config
```

Comment:
These notes should be turned into `/// @todo` so they can be more easily searched (that's also Doxygen-format which is widely understood by tooling)